geode-core/src/main/java/org/apache/geode/SystemFailure.java

```diff
@@ -142,7 +143,14 @@
  * </pre>
  *
  * @since GemFire 5.1
+ *
+ * @deprecated since Geode 1.11 since it is potentially counterproductive to try
```

Comment:
"since Geode 1.11 since" doesn't quite parse. Maybe "since Geode 1.11. It is potentially ...."

Maybe we could recommend people use -XX:+ExitOnOutOfMemoryError if they are starting geode processes without using gfsh? We already set that (actually the older option of -XX:OnOutOfMemoryError=kill -KILL %p) from gfsh start server.

Reply:
Improved the wording. Didn't add the recommendation about the JVM switch.